configure.ac

```diff
@@ -41,16 +41,23 @@ dnl *** misc complier/linker flags ***
 LIBTIMECODE_CFLAGS="-Wall"
 LIBTIMECODE_LDFLAGS="-export-symbols-regex '^(timecode_).*'"
 
-AC_MSG_CHECKING([if -Wl,--as-needed works])
-LFDLAGS_save=$LDFLAGS
-LDFLAGS="$LDFLAGS -Wl,--as-needed"
-AC_TRY_LINK([], [],
-            [
-             AC_MSG_RESULT([yes])
-             LIBTIMECODE_LDFLAGS="$LIBTIMECODE_LDFLAGS -Wl,--as-needed"
-            ], 
-            [AC_MSG_RESULT([no])])
-LDFLAGS=$LDFLAGS_save
+case $target_os in
+  *mingw32*|*win*)
+    LDFLAGS="-no-undefined"
+		;;
+  *)
+    AC_MSG_CHECKING([if -Wl,--as-needed works])
+    LDFLAGS_save=$LDFLAGS
+    LDFLAGS="$LDFLAGS -Wl,--as-needed"
+    AC_TRY_LINK([], [],
+                [
+                 AC_MSG_RESULT([yes])
+                 LIBLTC_LDFLAGS="$LIBTIMECODE_LDFLAGS -Wl,--as-needed"
```

Comment:
small copy/paste typo. should be 
LIBTIMECODE_LDFLAGS="$LIBTIMECODE_LDFLAGS ..."